pkg/migrator/migrator.go

```diff
@@ -156,7 +156,7 @@ func (m *migrator) deleteInstalledCscs(cscs []types.NamespacedName, operatorName
 func (m *migrator) deleteDatastoreCscs(operatorNamespace string) []error {
 	var errors []error
 	options := &client.ListOptions{}
-	options.SetLabelSelector(fmt.Sprintf("opsrc-datastore: \"true\""))
+	options.SetLabelSelector(fmt.Sprintf("opsrc-datastore"))
```

Comment:
Does this need to be hardcoded at all? Can we make this a const? I know at one time before we removed all this stuff it was part of the API package.

Reply:
oh duh

Reply:
good catch

Reply:
addressed